plugin/storage/es/esCleaner.py

```diff
@@ -60,13 +64,36 @@ def main():
 def filter_main_indices(ilo, prefix):
     ilo.filter_by_regex(kind='prefix', value=prefix + "jaeger")
     empty_list(ilo, "No indices to delete")
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-span-read'], exclude=True)
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-span-write'], exclude=True)
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-service-read'], exclude=True)
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-service-write'], exclude=True)
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-span-archive-read'], exclude=True)
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-span-archive-write'], exclude=True)
+    empty_list(ilo, "No indices to delete")
     # This excludes archive index as we use source='name'
     # source `creation_date` would include archive index
     ilo.filter_by_age(source='name', direction='older', timestring='%Y-%m-%d', unit='days', unit_count=int(sys.argv[1]))
 
 
+def filter_main_indices_rollover(ilo, prefix):
+    ilo.filter_by_alias(aliases=[prefix + 'jaeger-*-read'])
+    empty_list(ilo, "No indices to delete")
```

Comment:
Are these required after each filter, or just at the end? Does it cause problems calling the `filter_by_alias` on an empty list?

Reply:
Yes it's required otherwise it fails on empty cluster